src/extension/background-script/events/notifications.ts

```diff
@@ -22,7 +22,7 @@ const paymentSuccessNotification = async (
 
   const route = paymentResponseData?.data.route;
   const { total_amt, total_fees } = route;
-  const paymentAmount = total_amt - total_fees;
+  const paymentAmount = total_amt;
```

Comment:
I am not sure if this is correct for all the connectors? 
is that correct for LND, NWC, alby?

Reply:
alby to lndhub -correct
lndhub to alby - fees is always zero. total amount is correct no issues
lnd to lnd (testnet) - correct 
nwc to alby oauth - totalAmount correct, fees shown 0
alby oauth to nwc - totalAmount correct, fees shown 8 sats

Reply:
for all the cases totalAmount is correct which is the amount that user paid. but in some cases even if fees is charged fees is not returned in the route which maybe corrected on server side

Reply:
So I think LND returns the total_amt with the fees (which is why we deduct it here)
but all others return the paid amount without the fee.

which means your change is OK. but we might need to change the LND connector to return the sent amount.